README.rst

```diff
@@ -116,17 +68,24 @@ Make sure you have installed the following dependencies:
   * libpcap-dev
   * libssl-dev
   * libffi-dev
+  * libgmp-dev
+  * openssl
```

Comment:
`libssl-dev` is already in dependencies: https://github.com/TheTorProject/ooni-probe/pull/649#discussion-diff-84693609R69